sys-libs/mesa/mesa-17.1.10.recipe

```diff
@@ -55,13 +55,13 @@ PROVIDES_swpipe="
 	"
 REQUIRES_swpipe="
 	mesa$secondaryArchSuffix == $portVersion base
-	lib:libLLVM_7$secondaryArchSuffix
+	lib:libLLVM_9$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libz$secondaryArchSuffix
-	devel:libLLVM_7$secondaryArchSuffix
+	devel:libLLVM_9$secondaryArchSuffix
```

Comment:
There is still a llvm_config required below. Which one is that?

Reply:
cmd:llvm-config = llvm9-9.0.0-2